lib/matplotlib/rcsetup.py

```diff
@@ -307,6 +307,18 @@ def _validate_color_or_linecolor(s):
     raise ValueError(f'{s!r} does not look like a color arg')
 
 
+def validate_color_or_bool(c):
+    """Confirm c is a color or a bool."""
+    try:
+        return validate_color(c)
+    except ValueError:
+        pass
+    try:
+        return validate_bool(c)
+    except ValueError:
+        raise ValueError(f'Could not convert "{c}" to color or bool')
```

Comment:
```suggestion
        raise ValueError(f'Could not convert {c!r} to color or bool')
```
(Experience shows that people sometimes add a newline to the end of a string and this will make that case clear...)

Reply:
Although admittedly, it may not happen here...